### DISCUSSION TITLE
New Support Portal Access Links

### DISCUSSION THREAD

**Discussion by jonathonbyrdziak** [merchantprotocol/shadowmailer-support#2] [Announcements]
We organized the support system into two areas. 

1. This area on github is our feedback and communications area about all things shadowmailer.
2. The other is our documentation portal which is part of the shadowmailer system.

For the fastest response, post your questions, comments and support issues here to this portal. If you cannot find the documentation or don't know how to accomplish a task, please ask here in the discussions and we'll create the documentation for you.

**Comment by abadger13** [comment]
Hi -- I'm unable to install Poste.
I was getting errors in the ssh root@ip line in terminal. Failure to create mail server.
I was unable to fix this error.
I created a second server at Contabo. Went through the set up steps again. I get the same error "failure to create mail server".
I went back to the first IP server and now I get an error "remote host identification has changed". Terminal wants a host key verification. I don't know what this is.
Can't set up either server. Can't proceed with Poste.
Can you help please? Thanks.
Amy Biddle
